src/bp/migrations/v12_21_0-1618260392-bot_content_type_update.ts

```diff
@@ -9,10 +10,14 @@ const migration: Migration = {
   },
   up: async ({ bp, configProvider }: sdk.ModuleMigrationOpts): Promise<sdk.MigrationResult> => {
     const bots = await bp.bots.getAllBots()
+    const newTypes = ['builtin_video', 'builtin_audio']
+
     for (const [botId, botConfig] of bots) {
-      configProvider.mergeBotConfig(botId, {
-        imports: { contentTypes: [...botConfig.imports.contentTypes, 'builtin_video', 'builtin_audio'] }
-      })
+      if (_.difference(newTypes, botConfig.imports.contentTypes).length > 0) {
+        configProvider.mergeBotConfig(botId, {
+          imports: { contentTypes: [...botConfig.imports.contentTypes, ...newTypes] }
+        })
+      }
```

Comment:
I think you could simply use something like `_.merge` to make sure that both content-types are part of the config, but not duplicated. 

I know it's picky, but with this code, someone with a _botConfig.imports.contentTypes_ of `['builtin_audio']` would end up with `['builtin_audio', 'buildin_video', 'builtin_audio']`.

Reply:
Good point